### PR TITLE
Update testing limits for rpi4

### PR DESCRIPTION
**Pull request by cwfitzgerald**
**Connections**

Related to #1574. Exposed #1583.

**Description**

Changes the reftesting limits to be tolerant of how the RPI4 renders. Fixes the debug half of #1577. 

Fixes some naming issues as well.

**Testing**

It is

### CODE REVIEW

File: wgpu-hal/src/gles/egl.rs [hunk 1]
```diff
@@ -265,7 +265,8 @@ impl Inner {
             egl::CONTEXT_CLIENT_VERSION,
             3, // Request GLES 3.0 or higher
         ];
-        if flags.contains(crate::InstanceFlags::DEBUG) && !cfg!(target_os = "android") {
+        // Debug requires EGL 1.5+
+        if flags.contains(crate::InstanceFlags::DEBUG) && version.1 >= 5 {
```
Comment (kvark) [inline wgpu-hal/src/gles/egl.rs:269]: nit: we generally compare the whole `version` with the tuple

**Review by kvark — APPROVED** [review]
Thank you!
bors r+

File: wgpu-hal/src/gles/egl.rs (hunk 1 above)
Comment (cwfitzgerald) [reply wgpu-hal/src/gles/egl.rs]: Resolved

**Comment by cwfitzgerald** [comment]
bors retry

**Comment by cwfitzgerald** [comment]
borr r=kvark

**Comment by cwfitzgerald** [comment]
bors r=kvark

**Pull request merged**